.github/workflows/e2e-test.yaml

```diff
@@ -46,23 +46,30 @@ jobs:
         run: |
           curl --proto '=https' --tlsv1.2 -sSf https://tiup-mirrors.pingcap.com/install.sh | sh
           source /home/runner/.profile
-          tiup update --nightly
-          tiup playground nightly --tiflash=0 &
+          tiup install tidb:v4.0.5 tikv pd prometheus
+          tiup install playground
+          tiup update --all
+          tiup update --self
+          source /home/runner/.profile
```

Comment:
After using a fixed version maybe we no longer need any `tiup update`

Reply:
Yep, will remove it.